packages/docs/operations-manual/attestation-service.md

```diff
@@ -0,0 +1,311 @@
+# Attestation Service
```

Comment:
we'll also need to update the README for the attestation-service to either point to this or contain this new information.  it currently includes information about old configuration https://github.com/celo-org/celo-monorepo/tree/master/packages/attestation-service